docker-compose.yml

```diff
@@ -0,0 +1,21 @@
+version: "3.9"
+
+services:
+  postgresqldb:
+    image: postgres:latest
+    container_name: postgres
+    ports:
+      - 5432:5432
+    environment:
+      - POSTGRES_DB=expense_tracker_db
+      - POSTGRES_USER=postgres
+      - POSTGRES_PASSWORD=admin
+    restart: unless-stopped
+  flyway:
+    image: flyway/flyway
+    command: -url=jdbc:postgresql://postgresqldb:5432/expense_tracker_db -user=postgres -password=admin -connectRetries=60 migrate
```

Comment:
same

Reply:
same

Reply:
it's possible to put flyway credentials in the environment variables (https://flywaydb.org/documentation/configuration/parameters/). And for local use, you can create flyway.properties file and add it to gitignore (https://www.baeldung.com/database-migrations-with-flyway#3-external-configuration-file)